### DISCUSSION TITLE
Merge ReForge and Forge

### DISCUSSION THREAD

**Discussion by softtaco1** [lllyasviel/stable-diffusion-webui-forge#2442] [Ideas]
I want to start by saying I am not at all of the mindset that I am owed absolutely anything by anyone working on what is a passion project. I know the author of Forge @lllyasviel stays busy and likes to work on new projects and pushing things forward in a big way. With that said, there’s also lots of perks that continually come to ReForge that I’m a huge fan of. The main dev of ReForge is also a dev on Forge, @Panchovix . 

Would it be possible to have ReForge either as a direct branch to Forge, or, merge the two projects together? That way, while @lllyasviel is working on his big ideas, @Panchovix  can maintain steady progress in the interim. 

What do we think?

**Comment by flyingyakan** [comment]
reForge is a reliable environment for SDXL users who do not need flux support. reForge should remain an independent project.

**Reply by softtaco1** [reply]
Do you feel like Forge isn't a reliable environment for SDXL? I use it mainly for SDXL and find it incredibly reliable,  just lacking support for some newer models which would be fixed by merging Forge with reForge. 

Furthermore, having added support for separate models doesn't make Forge less reliable with others. At least, it hasn't in my experience.

**Comment by ScionoicS** [comment]
Hard to merge a gradio 3 and a gradio 4 codebase

**Reply by softtaco1** [reply]
That's a fair logistical obstacle. The other is that Gradio 5 is coming to Forge next year sometime too, I can see not wanting to put in all that effort just to have to redo a lot of it.
